### DISCUSSION TITLE
Idea for new interface 'Frequently made mistakes' on teachers page

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#1849] [Website]
I was taking a look at how I could improve the Hedy Docs (for teachers) and I think the main concern here is the 'frequently made mistakes' page. Of course I could do a lot myself, mainly updating the text, updating the fmm to the new level structure and improving the examples that are given. But it would also be really nice if it looked better then this long and messy list!

So I played around in powerpoint and came up with this possible lay-out. 

![Mockup Veelgemaakte fouten](https://user-images.githubusercontent.com/80678586/151701199-b61200b3-9be2-4aa3-a877-80779da807ba.png)

What do you think @Felienne and @TiBiBa ?

**Comment by TiBiBa** [comment]
Looks good to me! I can change the for-teachers structure to support this. We do have to make a clear differentiation for each level in the .yaml file. I'm not sure it is already structured for this.

**Reply by Felienne** [reply]
Yeah I think this is a great idea! Made an issue!
